plugins/doc_fragments/cmci.py

```diff
@@ -98,39 +108,85 @@ class ModuleDocFragment(object):
 options:
   resources:
     description:
-      - Options which specify a target resource
+      - Options that specify a target resource.
     type: dict
     required: false
     suboptions:   
-      criteria:
+      filter:
         description:
-          - A string containing logical expressions that filters the data 
-            returned on the request.
-          - The string that makes up the value of the CRITERIA parameter
-            follows the same rules as the filter expressions in the CICSPlex
-            SM application programming interface.
-          - The filter can work with options ``query``, ``update``, ``delete``; 
-            otherwise it will be ignored.
-          - For more guidance about specifying filter expressions using the
-            CICSPlex SM API, see
-            U(https://www.ibm.com/support/knowledgecenter/SSGMCP_5.6.0/system-programming/cpsm/eyup1a0.html).
+          - A string containing basic logical expressions that filter the resource table records
```

Comment:
I wonder if we need to say that the key is the attribute, and the value is the string to be filtered for. Eg NAME : 'PONG'